Controller/Adminhtml/Report/Save.php

```diff
@@ -202,9 +202,16 @@ public function updateVarnishConfiguration()
         $application_id = $this->state->getApplicationId();
         /** @var string $environment_name */
         $environment_name = $this->state->getEnvironmentName();
+        /** @var string $proxy_image*/
+        $proxy_image = $this->helper->getProxyImage($account_id, $application_id);
+
+        /** Extract the generated VCL code appropriate for their version*/
+        if (strpos($proxy_image, "4" !== false)){
```

Comment:
`strpos` would capture `5.4.2` or something similar?

Reply:
Since varnish 4.* needs one vcl file and all the other currently released versions(5.* and 6.*) need the other, I'm stripping off all digits but the major release. Varnish 5.4.2 is hitting this if-block as "varnish:5". Feel free to suggest if you see a better way.